2-Browsers/Week1/homework/ex4-whatsTheTime/index.js

```diff
@@ -8,7 +8,21 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/2-Brow
 2. Have the function execute when it's loading in the browser.
 ------------------------------------------------------------------------------*/
 function addCurrentTime() {
-  // TODO complete this function
-}
 
-// TODO execute `addCurrentTime` when the browser has completed loading the page
+  window.addEventListener('DOMContentLoaded', getCurrentTime);
+
+  const div = document.createElement('div');
+  document.body.appendChild(div);
+
+  function getCurrentTime() {
+    
+      setInterval(() => {
+        const date = new Date();
+        const time = date.getHours() + ":" + date.getMinutes() + ":" + date.getSeconds();
+        div.textContent = time
```

Comment:
You forgot to implement this part of the assignment
> Log the current time to the console also. Use setInterval() to make sure the time stays up to date.

```suggestion
        div.textContent = time
        // console.log()
```